schema/bom-1.6.xsd

```diff
@@ -54,6 +54,43 @@ limitations under the License.
         <xs:restriction base="bom:refType"/>
     </xs:simpleType>
 
+    <xs:simpleType name="version">
+        <xs:annotation>
+            <xs:documentation xml:lang="en"><![CDATA[
+                A single disjunctive version identifier, for a component or service.
+
+                Example values:
+                - "9.0.14"
+                - "v1.33.7"
+                - "7.0.0-M1"
+                - "2.0pre1"
+                - "1.0.0-beta1"
+                - "0.8.15"
+            ]]>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:restriction base="xs:normalizedString"/>
+    </xs:simpleType>
+    <xs:simpleType name="version-range">
```

Comment:
Same comment as above regarding lower camel case.

Reply:
thanks. fixed that.